hoodie-utilities/src/main/java/com/uber/hoodie/utilities/sources/AvroKafkaSource.java

```diff
@@ -31,17 +38,35 @@
 /**
  * Reads avro serialized Kafka data, based on the confluent schema-registry
  */
-public class AvroKafkaSource extends KafkaSource {
+public class AvroKafkaSource extends AvroSource {
+
+  private static volatile Logger log = LogManager.getLogger(AvroKafkaSource.class);
```

Comment:
Why do we need this to be volatile ? Do we expect this to be called and changed in different threads ?